src/datasource.ts

```diff
@@ -12,6 +12,7 @@ export const AppDataSource = new DataSource({
     },
     synchronize: true,
     logging: false,
+    timeTravelQueries: false,
```

Comment:
Was there a reason for disabling this?

Reply:
@dikshant If we enable timeTravelQueries, we were not able to perform any mutations on the db

Reply:
Ah right. AOST transactions are read only.